tools/torch_quant/torch_quant/graph.py

```diff
@@ -247,7 +247,16 @@ def quantizable_module_to_ref(ctx: GraphModContext) -> None:
                 f'module type {type(src)} is not supported for static quantization.')
         w_ob = src.qconfig.weight()
         w_ob(src.weight)
-        dst = dst_type.from_float(src, w_ob.qparams._asdict())
+        # update qscheme, since we don't have symmetric quant qscheme in quantized Tensor
+        # https://github.com/pytorch/pytorch/blob/v1.13.1/torch/ao/quantization/utils.py#L138
+        wq_dict = w_ob.qparams._asdict()
+        sym_to_aff_map = {
+            torch.per_tensor_symmetric: torch.per_tensor_affine,
+            torch.per_channel_symmetric: torch.per_channel_affine,
+        }
+        wq_dict['qscheme'] = sym_to_aff_map.get(wq_dict['qscheme'], wq_dict['qscheme'])
```

Comment:
The qscheme of weight is already `affine`, why should we add some logic to ensure this here.
And will there be any problems that use `symmetric` in calibration and change it to `affine` when convert the module to the ref one

Reply:
If backend is fbgemm or disc, the default weight qscheme is `per_channel_symmetric`. But symmetric quantized Tensor is not supported, so we should convert `symmetric` to `affine` here. Have we done this conversion before?
Generally, `symmetric` can be seen as a special case of `affine`. And the scales and zero points here are already certain values, the final result cannot be effected.

Reply:
If use DISC backend, we will use `torch.fake_quant_xxx`  instead of the really quantized tensor, so the `per_channel_symmetric ` is supported

Reply:
All right, while for DISC backend, we won't convert quantizable modules to ref version.